### DISCUSSION TITLE
Possibility to add custom templates to the init/new commands

### DISCUSSION THREAD

**Discussion by PashoQ** [sbt/sbt#7292] [Ideas]
Is there a way how to inject custom templates to the _sbt new_ or _sbt init_ commands via global plugins or some configuration?
We have several predefined templates in our team and would be nice to have them as part of the selection options of the init commands.

**Comment by eed3si9n** [comment]
When I first implemented it, the list was hardcoded, but here's a PR to make it more extensible - https://github.com/sbt/sbt/pull/7304
